Resources/config/dashboard.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="sonata.dashboard.cms_manager.dashboard.class">Sonata\DashboardBundle\CmsManager\CmsDashboardManager</parameter>
```

Comment:
Don't do that, according to https://symfony.com/doc/current/best_practices/business-logic.html#service-no-class-parameter